ansible_collections/arista/avd/galaxy.yml

```diff
@@ -51,3 +51,34 @@ homepage: https://avd.arista.com
 
 # The URL to the collection issue tracker
 issues: https://github.com/aristanetworks/avd/issues
+
+# Ignore the following files in the collection
+# overriding the default because we want some dotfiles.
+manifest:
+  directives:
+    - include meta/*.yml
+    - include COPYING LICENSE *.rst
+    - global-include *.txt *.md
+    - recursive-include tests **
+    - recursive-include roles **.yml **.yaml **.json **.j2 **.py **.gif **.png
+    - recursive-include playbooks **.yml **.yaml
+    - recursive-include changelogs **.yml **.yaml
+    - recursive-include plugins **.py **.yml **.yaml
+    # manifest.directives from galaxy.yml inserted here
```

Comment:
Not really clear in a year from now :) And since we also edited some of the generic ones above, it would be misleading to say these are our custom ones.

```suggestion
```